src/content/documentation/develop/developing-extensions-for-firefox-for-android-fennec.md

```diff
@@ -0,0 +1,408 @@
+---
+layout: sidebar
+title: Developing extensions for Firefox for Android (Fennec)
+permalink: /documentation/develop/developing-extensions-for-firefox-for-android-fennec/
+topic: Develop
+tags: [add-ons, beginner, guide, mobile, webextensions]
+contributors:
+  [caitmuenster, rebloor, juraj, mdnwebdocs-bot, ExE-Boss, Ding-Fan, andrewtruongmoz]
+last_updated_by: caitmuenster
+date: 2020-08-26
+---
+
+{% capture page_hero_banner_content %}
+
+# Developing extensions for Firefox for Android < version 68
+
+::: note alert
+This article discusses developing extensions for Firefox for Android prior to version 68 (codenamed "Fennec"). Fennec is in maintenance mode as Mozilla rolls out a new, reimagined mobile browsing experience (codenamed "Fenix"). For more information about developing extensions for Fenix, please see [this article](/documentation/develop/developing-extensions-for-firefox-for-android/).
+:::
+
+You'll approach the coding of an extension for Firefox for Android in the same way as you would for a desktop extension; using a text editor or tool of your choice to write the code. However, when you want to test and debug your extension you need to follow a different process, this article walks you through that process.
+
+Using [web-ext](/documentation/develop/getting-started-with-web-ext/) in your extension development is recommended. Follow the set up and debugging processes described here, but use [`web-ext run`](/documentation/develop/web-ext-command-reference#web-ext-run) to execute your extension on Firefox for Android. Among other advantages, using [web-ext](/documentation/develop/getting-started-with-web-ext/) automatically restarts your extension on Firefox for Android when you make edits. Also, you can take advantage of [`web-ext lint`](/documentation/develop/web-ext-command-reference/#web-ext-lint), which performs a check to determine if any of the permissions, manifest keys, and web extension APIs you’re using are incompatible with Firefox for Android.
+
+However, instructions are provided for the steps you need to take should you choose not to use [web-ext](/documentation/develop/getting-started-with-web-ext/).
+
+{% endcapture %}
+{% include modules/page-hero.liquid
+    content: page_hero_banner_content
+%}
+
+<!-- END: Page Hero Banner -->
+
+<!-- Content with Table of Contents Module -->
+
+{% capture content_with_toc %}
+
+## Set up your computer and Android emulator or device
+
+::: note alert
+This section only applies to Firefox for Android < 68 (Fennec)
```

Comment:
Would it be more helpful to link to the latest do c from here? Or at least a pointer to the top of the doc that explains the deprecation+ correct doc location?

Reply:
Hey @Rob--W, I'm not sure what you mean by "do c." But I did see that a bunch of stuff got messed up when I copied over the old Fennec page, so I'm going to fix that!

Reply:
Sorry, I typed that on mobile.

I meant "doc", to reference the most up to date documentation from here so that people who unknowingly arrived at the wrong page can quickly find the latest docs.

Reply:
Yeah! I removed the warnings from the individual sections -- those were carry overs from when I tried to consolidate all information on one page -- and am just putting one warning at the top of the page with a link to the Fenix documentation. Can you take a look and let me know if that looks ok?